### ISSUE TITLE
armory tx broadcast timed out

### ISSUE THREAD

**Issue by V-ICE** [goatpig/BitcoinArmory#169]
[armorylog_20170210_1005.txt](https://github.com/goatpig/BitcoinArmory/files/766162/armorylog_20170210_1005.txt)

Trying to send btc but always getting tx timed out. Raised the Fee, but same error

**Comment by goatpig** [comment]
Broadcast it manually, you'll get an error message from your node. That should help you figure it out.

**Comment by V-ICE** [comment]
Allright, how do I do that ?

**Comment by V-ICE** [comment]
Tried to take the signed hex to coinb.in to broadcast it, but there I get "missing inputs"
Tried with Electum to broadcast it there. But The error is Missing inputs there too.
Please Advise

**Comment by 2SI3NX** [comment]
actually im having the exact same issue.  @V-ICE did you fix it?  Ive tried sending bitcoins 4 times with the same error message.

**Comment by V-ICE** [comment]
@2si3nx Yes, after a couple of reinstalld and weeks of downloads and syncing I was able to send my bitcoins. It had to do with Armory reported the blockchain sync completed. But in the realeorld it was NOT. After both PC -and Linux installs I got to trnsfer the last.  Bitcoins from armory to another more reliable and stable wallet.

**Comment by 2SI3NX** [comment]
@V-ICE may i ask which wallet you're using now?

**Comment by V-ICE** [comment]
@2SI3NX Using Greenwallet https://greenaddress.it/

**Comment by keithlovett** [comment]
This is getting silly. As soon as I can get my bitcoin out of Armory I'll be transferig the lot to a more stable wallet too. Just gotta get Armory to allow me to send my coins!!??

**Comment by goatpig** [comment]
Did you update?

**Comment by keithlovett** [comment]
This is beyond silly. I've tried so many different configurations of Bitcoin Core and Armoury. I've lost all faith  and now just want to move my coins to a stable wallet.

I'm going to try updating to v0.96.0.4-testing but don't hold up much hope. Can someone please explain clearly how we can access coins. It's been well over a month since I managed to access them myself.

Will let you know what happens after update.
